src/sdrf_pipelines/sdrf/schemas/utils.py

```
@@ -0,0 +1,132 @@
+
+def _merge_fields_last_strategy(merged: ColumnDefinition, col_def: ColumnDefinition) -> None:
+    """Merge fields using LAST strategy - use last non-null value."""
+    for field in ["description", "requirement", "allow_not_applicable", "allow_not_available"]:
+        if getattr(col_def, field) is not None:
+            setattr(merged, field, getattr(col_def, field))
+
+
+def _merge_fields_combine_strategy(merged: ColumnDefinition, col_def: ColumnDefinition) -> None:
+    """Merge fields using COMBINE strategy - combine descriptions and use stricter requirement level."""
+    # Merge description field
+    if getattr(col_def, "description") and getattr(col_def, "description") not in (None, ""):
```

Ruff (0.14.14)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)